### PR TITLE
config templating added using handlebar

### PR DESCRIPTION
**Pull request by avik-sengupta-kong**
config templating added using handlebar

### CODE REVIEW

File: config/workspace-data.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "ws": [
+      {
+        "name": "hsbc-zeta",
```
Comment (srb3) [inline config/workspace-data.json]: minor point: public repo lets not use company names

File: config/workspace-data.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            "rate-limiting-advanced-limit": [10,20],
+            "rate-limiting-advanced-window": [60,120],
+            "rate-limiting-advanced-redis-host": "redis.kong-hybrid-dp.svc.cluster.local",
+            "rate-limiting-advanced-redis-port": "6379",
+            "correlation-id-header-name":"Kong-Request-ID"
+
+        }
+      },
+      {
+        "name": "hsbc-eta",
```
Comment (srb3) [inline config/workspace-data.json]: same here

File: app.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const yaml = require('js-yaml');
+const handlebars= require('handlebars');
```
Comment (srb3) [inline app.js]: shall we include a space before the '=' to keep it consistent

File: app.js [hunk 4]
```diff
@@ -17,12 +18,33 @@ const https = require("https");
     var pluginsEndpoint = '/plugins';
 
     let configDir = './config/workspaces.yaml';
+    let configHbs='./config/workspace-template.hbs'
```
Comment (srb3) [inline app.js]: consistent spacing

**Review by srb3 — CHANGES_REQUESTED** [review]

**Comment by avik-sengupta-kong** [comment]
All requested changes addressed.

**Comment by avik-sengupta-kong** [comment]
No longer relevant.

**Pull request closed without merging**